src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="home-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet">
+  </head>
+  <body class="home-page__content">
+    <header class="container">
+      <img class="image" src="/images/imac.jpeg"
+        alt="APPLE A1419 iMac27">
```

Comment:
don't forget about code formatting. write it in one line to start each attribute from new line
```suggestion
      <img
        class="image"
        src="/images/imac.jpeg"
        alt="APPLE A1419 iMac27"
      >
```